hr_employee_age/models/hr_employee.py

```diff
@@ -27,7 +10,6 @@ class HrEmployee(models.Model):
 
     age = fields.Integer(
         string='Age',
-        readonly=True,
```

Comment:
Why you've decided to remove the `readonly` flag? It's age a definite function of birthday?

Reply:
@alexey-pelykh, Compute field will be by default read-only, so no need to define readonly=True inside the field definition. I have tested after the remove this attribute. It's working well.